chroma-agent/chroma_agent/action_plugins/manage_targets.py

```diff
@@ -115,7 +115,13 @@ def get_resource_locations():
     """Parse `crm_mon -1` to identify where (if anywhere)
        resources (i.e. targets) are running."""
 
-    rc, lines_text, stderr = AgentShell.run_old(["crm_mon", "-1", "-r"])
+    try:
+        rc, lines_text, stderr = AgentShell.run_old(["crm_mon", "-1", "-r"])
+    except OSError, e:
+        # ENOENT is fine here.  Pacemaker might not be installed yet.
+        if e.errno != errno.ENOENT:
```

Comment:
Sure, that wanders into the configuration management realm, which I think we want to tackle at some point, but not just yet.